articles/tech-resources/custom-bounce-domain.md

```diff
@@ -4,26 +4,50 @@ redirect_from: "https://support.sparkpost.com/customer/portal/articles/2371794-c
 description: "You can improve your deliverability and email branding by adding a custom bounce domain A custom bounce domain provides another layer to the branding that some IS Ps care about Note Only one custom bounce domain is permitted per account at this time By adding a bounce domain to your..."
 ---
 
-You can improve your deliverability and email branding by adding a custom bounce domain. A custom bounce domain provides another layer to the branding that some ISPs care about.
+You can improve your deliverability and email branding by adding a bounce domain to each of your sending domains. A bounce domain provides another layer to the branding that some ISPs care about.
```

Comment:
This is how it was in the google doc, but reading it more closely now this doesn't make sense.  

I think we need to change this to:

You can improve your deliverability and email branding by configuring each of your sending domains as bounce domains. A bounce domain provides another layer to the branding that some ISPs care about.